### DISCUSSION TITLE
Change the email verification URL endpoint

### DISCUSSION THREAD

**Discussion by droslean** [keycloak/keycloak#28208] [Q&A]
Hello

Keycloak is generating a URL for email verification like: `https://mydomain.com/realms/myrealm/login-actions/action-token?key=$KEY`

Is there any way to change it?

**Comment by droslean** [comment]
Related code: https://github.com/keycloak/keycloak/blob/main/services/src/main/java/org/keycloak/authentication/requiredactions/VerifyEmail.java#L131-L160

**Comment by ghost** [comment]
I also have the same question. Did you figure it out @droslean?

**Reply by droslean** [reply]
> I also have the same question. Did you figure it out @droslean?

@joao-moonward The only way is to write your own Java code snippet to implement a custom email verifier.

**Reply by ghost** [reply]
Thanks mate
